src/dials/algorithms/indexing/stills_indexer.py

```diff
@@ -702,6 +692,9 @@ def identify_outliers(self, params, experiments, indexed):
 
         RR = refiner.predict_for_reflection_table(indexed)
 
+        if params.refinement.reflections.outlier.algorithm != "sauter_poon":
+            return refiner.selection_used_for_refinement()
```

Comment:
Does this selection include any reflections that were subsequently filtered during refinement?